pkg/kv/kvserver/idalloc/id_alloc.go

```diff
@@ -91,10 +92,9 @@ func (ia *Allocator) start() {
 			var res kv.KeyValue
 			for r := retry.Start(base.DefaultRetryOptions()); r.Next(); {
 				idKey := ia.idKey.Load().(roachpb.Key)
-				if err := ia.stopper.RunTask(ctx, "storage.Allocator: allocating block", func(ctx context.Context) {
+				if stopperErr := ia.stopper.RunTask(ctx, "idallocator: allocating block", func(ctx context.Context) {
```

Comment:
`s/idallocator/idalloc/`? to remain consistent with the package name?

Reply:
done